quinn-proto/src/connection/streams/state.rs

```diff
@@ -415,6 +417,7 @@ impl StreamsState {
         pending: &mut Retransmits,
         retransmits: &mut ThinRetransmits,
         stats: &mut FrameStats,
+        #[allow(unused)] qlog: &mut QlogSentPacket,
```

Comment:
Would this be better as `#[cfg(feature = "qlog")]`?  Or is there some reason that this doesn't work?